conda/cli/conda_argparse.py

```diff
@@ -134,6 +134,7 @@ def error(self, message):
                             from ..exceptions import CommandNotFoundError
                             raise CommandNotFoundError(cmd)
                         args = [find_executable('conda-' + cmd)]
+                        args.extend(sys.argv[2:])
```

Comment:
This fixes a bug that's a show-stopper for 4.4.5.  Meaning 4.4.5 will never be released to defaults.

Reply:
Just an error/omission in copying over the code from 4.3.x.